codecov.yml

```diff
@@ -2,11 +2,11 @@ comment: false
 
 coverage:
   status:
-    patch:
```

Comment:
Got it. 10% seems reasonably quiet.

I wonder if `patch` should still have a target of zero, otherwise PRs will fail if they don't meet the overall project coverage. It's not uncommon to add or change uncovered code in a PR.